snippets/csharp/keywords/StackAllocExamples.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace keywords
+{
+    public static class StackAllocExamples
+    {
+        public static void Examples()
+        {
+            FibonacciGeneration();
+            Console.WriteLine("=========================");
+            InitializeBitMasks();
+        }
+
+        private static unsafe void InitializeBitMasks()
+        {
+            // <Snippet2>
+            int* mask = stackalloc[] {
+                0b0000_0000_0000_0001,
+                0b0000_0000_0000_0010,
+                0b0000_0000_0000_0100,
+                0b0000_0000_0000_1000,
+                0b0000_0000_0001_0000,
+                0b0000_0000_0010_0000,
+                0b0000_0000_0100_0000,
+                0b0000_0000_1000_0000,
+                0b0000_0001_0000_0000,
+                0b0000_0010_0000_0000,
+                0b0000_0100_0000_0000,
+                0b0000_1000_0000_0000,
+                0b0001_0000_0000_0000,
+                0b0010_0000_0000_0000,
+                0b0100_0000_0000_0000,
+                0b1000_0000_0000_0000
```

Comment:
nit: would adding one more underscore to literals make it more readable, e.g., `0b_1000_0000_0000_0000`?